src/pkg/logger/logger.go

```diff
@@ -37,6 +37,22 @@ var (
 	Error = Level(slog.LevelError) // 8
 )
 
+// String returns the string representation of the Level.
+func (l Level) String() string {
```

Comment:
This is a fine step for now but if we end up needing anything more complex in the future, we should consider making Level a complex type. e.g.

```
type Level struct {
    Value int
    s string
} 

func (l Level) String() string {
    return s
}

// todo: Equal(level Level) bool {...} ?

var (
    Debug = Level{slog.LevelDebug, "debug"}
    ...
)
```

There's other valid ways to approach this too, just food for thought!

Reply:
Yeah fair